base/remote/R/remote.copy.from.R

```diff
@@ -45,5 +45,37 @@ remote.copy.from <- function(host, src, dst, options = NULL, delete = FALSE, std
     }
   }
   PEcAn.logger::logger.debug("rsync", shQuote(args))
-  system2("rsync", shQuote(args), stdout = TRUE, stderr = as.logical(stderr))
+  out <-
+    system2("rsync", args, stdout = "", stderr = as.logical(stderr))
```

Comment:
is removing the `shQuote` around `args` intentional?

Reply:
Yes, it was intentional, but now that you bring it up I'm not sure it's correct.  `system2()` always `shQuote`s `command`, but I guess it doesn't do that for `args`.  I'll add it back and make sure everything still works.